sqld/src/main.rs

```diff
@@ -21,7 +21,7 @@ static GLOBAL: MiMalloc = MiMalloc;
 /// SQL daemon
 #[derive(Debug, Parser)]
 #[command(name = "sqld")]
-#[command(about = "SQL daemon", version, long_about = None)]
+#[command(about = "SQL daemon", long_about = None)]
```

Comment:
https://docs.rs/clap/latest/clap/_derive/index.html#command-attributes `version` can take an expression parameter, no need to add a new argument